docs/releases/v5_0_0_3001-x86_64.md

```diff
@@ -0,0 +1,25 @@
+# libhkl-v5.0.0.3001-x86_64.tar.gz
+
+[v1.1.0](https://github.com/bluesky/libhkl/releases/tag/v1.1.0) | `libhkl-v5.0.0.3001-x86_64.tar.gz`
+
+package | version
+--- | ---
+sha256 | `b0a79417855ab358a7f1c5e08074b3e7e014b5d2e10c325d6e1328f3442bf69d`
+built on | [Docker](https://github.com/bluesky/libhkl/tree/main/builder)
+architecture | x86_64
+OS | debian:bullseye
+`gobject-introspection` | 1.78.1 (built from source)
+`libgsl-dev` | 2.7 (built from source)
+
+Requirements to build a conda-forge package:
+
+```text
+    gobject-introspection =1.78.1
+    gsl =2.7
+    pygobject
+    python >=3.8,<3.12
+```
+
+NOTE: `gobject-introspection =1.64` is available on conda-forge for py36-py38.
+To get later versions and use contemporary Python versions, need to build these
+somewhere else than GitHub Actions workflows.
```

Comment:
Should this note go in the other release notes as well?

Reply:
I'm leaving that one as historical comment.

Reply:
Added similar notes to the others.